ui-tests/src/test/java/io/syndesis/qe/pages/integrations/list/IntegrationByStatus.java

```diff
@@ -0,0 +1,7 @@
+package io.syndesis.qe.pages.integrations.list;
+
+/**
+ * Created by sveres on 11/16/17.
+ */
+public class IntegrationByStatus {
```

Comment:
I'm not sure if there is purpose for this class. Is there?

Reply:
sorry, this one is here by mistake